qmetaobject/Cargo.toml

```
@@ -12,9 +12,10 @@ keywords = ["Qt", "QML", "QMetaObject",]
 repository = "https://github.com/woboq/qmetaobject-rs"
 
 [features]
-default = ["log"]
+default = ["log", "widgets"]
 chrono_qdatetime = ["qttypes/chrono"]
 webengine = ["qttypes/qtwebengine"]
+widgets = ["qttypes/qtwidgets"]
 
 [dependencies]
 qttypes = { path = "../qttypes", version = "0.2.5", features = ["qtquick"] }
```

```suggestion
qttypes = { path = "../qttypes", version = "0.2.5", default-features = false, features = ["required", "qtquick"] }
```

This PR is extremely helpful for me: I'm creating an app for the reMarkable e-ink tablet, which only supports QtQuick. However, `qttypes` still tries to link to QtWidgets with its default features, which causes an error for me because it doesn't exist.